woocommerce/woocommerce
# Add fallback image payments task gateway icons

### Changes proposed in this Pull Request:

Closes https://github.com/woocommerce/woocommerce/issues/32575.

### How to test the changes in this Pull Request:

1. In your local, set store country to US and don't select WCPay during onboarding
2. Make sure Stripe plugin isn't set up
3. Hardcode [this condition](https://github.com/woocommerce/woocommerce/blob/af2349c6859b9f602852aa4511e064e08d8b4f70/plugins/woocommerce/src/Admin/Features/PaymentGatewaySuggestions/Init.php#L66-L68) to true so that it'll load local gateway config
4. Change [Stripe 72x72 image URL](https://github.com/woocommerce/woocommerce/blob/af2349c6859b9f602852aa4511e064e08d8b4f70/plugins/woocommerce/src/Admin/Features/PaymentGatewaySuggestions/DefaultPaymentGateways.php#L43) to an non-existent URL, such as `stripe1234.png`
5. Go to Set up payments task
6. Observe Stripe gateway image loads the fallback image since the request is returning 404

<img width="794" alt="image" src="https://user-images.githubusercontent.com/3747241/165220443-20fbcb2c-389a-47ed-9f88-1b9282183604.png">




### FOR PR REVIEWER ONLY:
